keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageService.kt

```diff
@@ -183,18 +193,23 @@ class ImageService(
  *
  * The resulting map will contain no entry for regions where an image is not found. The calling
  * code must check this if it requires all regions to be present.
+ *
+ * @param baseOs if supplied only images with that base OS are considered.
  */
 suspend fun ImageService.getLatestNamedImages(
   appVersion: AppVersion,
   account: String,
-  regions: Collection<String>
+  regions: Collection<String>,
+  // TODO: make this non-nullable
+  baseOs: String? = null
 ): Map<String, NamedImage> = coroutineScope {
```

Comment:
Added a new `baseOs` param here. There are other places this method is used but I think all of them should specify a `baseOs`, hence the `TODO` comment